docs/QUnit/only.md

```diff
@@ -8,7 +8,7 @@ categories:
 
 ## `QUnit.only( name, callback )`
 
-Adds a test to exclusively run, preventing all other tests from running.
+Add a set of tests to exclusively run, preventing other tests from running.
```

Comment:
This sounds like it allows you to add a set of tests with a single invocation. Maybe it could instead say:

```markdown
Adds a test to exclusively run, preventing any other tests not defined with `QUnit.only()` from running
```

Reply:
Updated.